sdk/servicebus/service-bus/review/service-bus.api.md

```diff
@@ -419,23 +423,23 @@ export interface ServiceBusReceivedMessage extends ServiceBusMessage {
 export interface ServiceBusReceiver {
     abandonMessage(message: ServiceBusReceivedMessage, propertiesToModify?: {
         [key: string]: any;
-    }): Promise<void>;
+    }, options?: OperationOptionsBase): Promise<void>;
     close(): Promise<void>;
-    completeMessage(message: ServiceBusReceivedMessage): Promise<void>;
+    completeMessage(message: ServiceBusReceivedMessage, options?: OperationOptionsBase): Promise<void>;
     deadLetterMessage(message: ServiceBusReceivedMessage, options?: DeadLetterOptions & {
```

Comment:
`OperationOptionsBase` is missing for deadletter, please add.

Individual tests for settlement methods are required to make sure of this.